src/screens/menu/MenuScreen.tsx

```diff
@@ -0,0 +1,149 @@
+import {getExposedHelpMenuURL} from 'shared/RegionLogic';
+import {captureException} from 'shared/log';
+import {useAccessibilityAutoFocus} from 'shared/useAccessibilityAutoFocus';
+
+import {ConditionalMenuPanels} from './views/ConditionalMenuPanels';
+import {PrimaryMenuButtons} from './views/PrimaryMenuButtons';
+import {InfoShareItem} from './components/InfoShareItem';
+import {CloseButton} from './components/CloseButton';
+
+export const MenuScreen = () => {
+  const [systemStatus] = useSystemStatus();
+  const navigation = useNavigation();
+  const i18n = useI18n();
+  const {region} = useStorage();
+  const regionalI18n = useRegionalI18n();
+
+  const onPrivacy = useCallback(() => {
+    Linking.openURL(i18n.translate('Info.PrivacyUrl')).catch(error => captureException('An error occurred', error));
+  }, [i18n]);
+
+  const onGetCode = useCallback(() => navigation.navigate('NoCode'), [navigation]);
+  const onLearnMore = useCallback(() => navigation.navigate('Tutorial'), [navigation]);
+  const onLanguage = useCallback(() => navigation.navigate('LanguageSelect'), [navigation]);
+  const onRegion = useCallback(() => navigation.navigate('RegionSelect'), [navigation]);
+  const onHelp = useCallback(() => {
+    Linking.openURL(i18n.translate('Info.HelpUrl')).catch(error => captureException('An error occurred', error));
+  }, [i18n]);
+
+  const regionIcon = region !== undefined && region !== 'None' ? 'icon-external-arrow' : 'icon-chevron';
+
+  const onExposedHelp = useCallback(() => {
+    if (region !== undefined && region !== 'None') {
+      Linking.openURL(getExposedHelpMenuURL(region, regionalI18n)).catch(error =>
+        captureException('An error occurred', error),
+      );
+    } else {
+      navigation.navigate('RegionSelectExposedNoPT', {drawerMenu: true});
+    }
+  }, [navigation, region, regionalI18n]);
+
+  const versionNumber = `${i18n.translate('OverlayOpen.Version')}: ${APP_VERSION_NAME} (${APP_VERSION_CODE})`;
+  const autoFocusRef = useAccessibilityAutoFocus(true);
+
+  return (
+    <SafeAreaView style={styles.flex}>
+      <Box backgroundColor="overlayBackground" paddingHorizontal="m" flex={1}>
+        <Box flexDirection="row">
+          <Box flex={1} style={styles.title} paddingRight="m">
+            <Text paddingVertical="m" focusRef={autoFocusRef}>
+              <StatusHeaderView enabled={systemStatus === SystemStatus.Active} />
+            </Text>
+          </Box>
+          <Box marginVertical="m">
+            <CloseButton />
+          </Box>
+        </Box>
+        <ScrollView style={styles.flex}>
+          <Box marginBottom="m">
+            <PrimaryMenuButtons />
+          </Box>
+
+          <Box marginBottom="m">
+            <ConditionalMenuPanels />
+          </Box>
+
+          <Box marginBottom="m" testID="InfoShareViewID">
+            <InfoShareItem
+              testID="getCodeButton"
+              onPress={onGetCode}
+              text={i18n.translate('Info.GetCode')}
```

Comment:
Minor - but what if we moved the `text attr` to the top of all the  `InfoShareItem` components to make it a little easier to see what the item actually is.

Reply:
good idea, I will do that